src/sdk/result_set_sql.h

```diff
@@ -91,20 +96,28 @@ class ResultSetSQL : public ::hybridse::sdk::ResultSet {
 
     int32_t Size() override { return result_set_base_->Size(); }
 
+    void SetReadableTime(bool readable_time) {
+        readable_time_ = readable_time;
+    }
+
+    const bool GetAsString(uint32_t idx, std::string& val) override;
+
  private:
     ::hybridse::vm::Schema schema_;
     uint32_t record_cnt_;
     uint32_t buf_size_;
     std::shared_ptr<brpc::Controller> cntl_;
     ResultSetBase* result_set_base_;
     std::shared_ptr<butil::IOBuf> io_buf_;
+    bool readable_time_;
```

Comment:
it's better that`ResultSet` should know nothing about readable time ?

Where prettify timestamp & date (or any other type) as string, appears in `sql_cmd.h`, with the customized data type formater provided